### PR TITLE
README: update distrobox section to include alternative directions 

### PR DESCRIPTION
**Pull request by spaceguybob**
…r the distrobox binary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ curl -s https://raw.githubusercontent.com/89luca89/distrobox/main/install | sh -
 sudo mv ~/.local/share/distrobox* /usr/lib/apx
 ```
 
+> In the above command, you can replace the path with: `/usr/local/lib/apx/`if preferred. You will also need to edit `/etc/apx/config.json` to look for distrobox in `/usr/local/lib/apx/`(This is required to install apx on some immutable distros that use OStree)
```
Comment (MonsterObserver) [inline README.md:114]:
```suggestion
> In the above command, you can replace the path with: `/usr/local/lib/apx/` if preferred. You will also need to edit `/etc/apx/config.json` to look for distrobox in `/usr/local/lib/apx/` (This is required to install apx on some immutable distros that use OSTree, such as Silverblue)
```

**Comment by spaceguybob** [comment]
Thank u

**Comment by kbdharun** [comment]
Thanks for your contribution, I am closing this PR as the README has been updated to reflect a similar approach for specifying different directories in make.

**Comment by spaceguybob** [comment]
> Thanks for your contribution, I am closing this PR as the README has been updated to reflect a similar approach for specifying different directories in make.

thats fine, i was unaware of that shift at the time

**Pull request closed without merging**